src/plugins/intel_gpu/tests/unit/test_cases/convolution_gpu_test.cpp

```diff
@@ -7927,7 +7927,9 @@ TEST_P(convolution_grouped_gpu, base) {
     auto outputs = network.execute();
 
     auto out_mem = outputs.at("conv").get_memory();
-    cldnn::mem_lock<float> out_ptr(out_mem, get_test_stream());
```

Comment:
`cldnn::mem_lock` uses `mem_lock_type::read_write` access modifier by default, I think all these changes can be simplified by applying `mem_lock_type::read` - then implicit memory copy call to host will be performed

Reply:
Good tip! Thanks!